lib/screens/v2/screen_data/parameters.ex

```diff
@@ -38,7 +38,7 @@ defmodule Screens.V2.ScreenData.Parameters do
     },
     elevator_v2: %Static{
       candidate_generator: CandidateGenerator.Elevator,
-      refresh_rate: 30
+      refresh_rate: 8
```

Comment:
Huh, is our "persistent" component logic actually tied to the data refresh rate? That seems less than ideal, but maybe just a refactoring opportunity to keep in mind for the future. (Suppose your screen consists almost entirely of a paging component, and you expect it to normally have at least two pages: it can only "really" refresh once all the paging is done, so you're doing at least twice as many data refreshes as you need to.)

Reply:
It does currently, yes, and I just felt like it would be too far out of scope to adjust it here. Given that data refreshes are built into the framework, we may need to get around it by preventing it from refreshing at this level and adding data fetches to the widget component itself. Then where we call `onFinish()` currently could be replaced by that fetch.